main.css

```diff
@@ -9,8 +9,8 @@ body {
   background-color: #fff;
 }
 
-/* header section */
-.header-section {
+/* toolbar section */
+.toolbar-section {
   display: flex;
   flex-direction: row;
   column-gap: 17em;
```

Comment:
| Original      | Yours |
  | ----------- | ----------- |
|<img width="321" alt="Screen Shot 2021-11-24 at 1 16 53 PM" src="https://user-images.githubusercontent.com/69011963/143200429-638f1328-1b95-460f-aa18-827a884990dd.png">|![Screen Shot 2021-11-24 at 12 52 52 PM](https://user-images.githubusercontent.com/69011963/143200509-3fd9ff4c-0424-4cbb-9d28-ab65f86286ec.png)|

- [ ] Also, your social icons are not showing up on the page. Please try to fix this issue as well. 😊